pkg/provider/bitbucketserver/bitbucketserver.go

```diff
@@ -31,13 +39,38 @@ type Provider struct {
 }
 
 func (v *Provider) Validate(ctx context.Context, params *params.Run, event *info.Event) error {
+	signature := event.Request.Header.Get("X-Hub-Signature")
```

Comment:
I think we can just reuse the function from the github library (and even the full fuction since it has the same X-Header)

Reply:
thanks for the change, did you have  a chance to test manually ? since we don't have e2e tests on bitbucket server

Reply:
yes works fine 🙃